src/mimepart.cpp

```diff
@@ -31,7 +31,13 @@ MimePart::MimePart() : d_ptr(new MimePartPrivate)
 
 MimePart::MimePart(const MimePart &other)
 {
-    qDebug() << Q_FUNC_INFO;
+    setCharset(other.charset());
```

Comment:
It would be better to set them directly using the d pointer, not sure compiler would optimize this.
While you are at it, it seem the setContent is leaking the QBuffer and it's d_ptr, could you please verify?
thanks

Reply:
Ok, I will take a look at all requests and commit it tomorrow then.

Reply:
Ok, I just checked and it doesn't leak, I forgot it used QSharedPointer